File: app/src/main/java/com/example/asthmaapplication/main/mainpage/subpages/QuizzesExamsFragment.java
package com.example.asthmaapplication.main.mainpage.subpages;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.Nullable;
import androidx.lifecycle.ViewModelProvider;

import com.example.asthmaapplication.R;
import com.example.asthmaapplication.databinding.FragmentQuizzesExamsBinding;
import com.example.asthmaapplication.main.common.BaseFragment;
import com.example.asthmaapplication.main.mainpage.MainViewModel;

import javax.inject.Inject;

import io.reactivex.annotations.NonNull;

public class QuizzesExamsFragment extends BaseFragment {
    MainViewModel viewModel;
    FragmentQuizzesExamsBinding binding;

    @Inject
    public QuizzesExamsFragment() {

    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        viewModel = new ViewModelProvider(this).get(MainViewModel.class);
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @javax.annotation.Nullable ViewGroup container, @javax.annotation.Nullable Bundle savedInstanceState) {
        binding = FragmentQuizzesExamsBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    @Override
    public void onViewCreated(@Nullable View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        getPreferences();
        setActionBarTitle();
    }

    @Override
    public void onResume() {
        super.onResume();
    }

    public void setActionBarTitle() {
        setActionBarTitle(getString(R.string.quizzes_exam_page_title));
    }

    @Override
    public View getRoot() {
        return binding.getRoot();
    }
}

File: app/src/main/java/com/example/asthmaapplication/main/mainpage/subpages/learningbasepages/QuizzesExamsFragment.kt
package com.example.asthmaapplication.main.mainpage.subpages.learningbasepages

import javax.inject.Inject
import com.example.asthmaapplication.main.common.BaseFragment
import com.example.asthmaapplication.main.mainpage.MainViewModel
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.inventory.R
import com.example.inventory.databinding.FragmentQuizzesExamsBinding
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class QuizzesExamsFragment @Inject constructor() : BaseFragment() {
    var viewModel: MainViewModel? = null
    var binding: FragmentQuizzesExamsBinding? = null


    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentQuizzesExamsBinding.inflate(inflater, container, false)
        return binding!!.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view!!, savedInstanceState)
        setActionBarTitle()
    }

    override fun onResume() {
        super.onResume()
    }

    fun setActionBarTitle() {
        setActionBarTitle(getString(R.string.quizzes_exam_page_title))
    }

    override fun getRoot(): View {
        return binding!!.root
    }
}